### ISSUE TITLE
NVDA does not announce the change of status of certain HTML elements in Microsoft Edge

### ISSUE THREAD

**Issue by brunoprietog** [nvaccess/nvda#13316]
### Steps to reproduce:

Press the button below

<details>
<summary>Press the button</summary>
It doesn't say anything
</details>

### Actual behavior:

NVDA should have reported that the button was expanded

This happens in other cases as well, for example, in a select that, when changing from one option to another, this shows/hides an element in the dom, and NVDA does not announce the change of the select that is focused.

### Expected behavior:

NVDA does not report anything

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

NVDA 2021.3.1

#### Windows version:

Windows 11 21H2 (x64) build 22000.469

#### Name and version of other software in use when reproducing the issue:

Microsoft Edge 97.0.1072.76

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes

**Comment by LucyLiYY** [comment]
The issue also repro on the environment below, NVDA doesn't announce the expand/collapsed status change.
Microsoft Edge
Version 102.0.1245.33 (Official build) (64-bit)
NVDA
Version: 2021.3.5

**Comment by Adriani90** [comment]
This works as expected now with NVDA last alpha and Edge and Chrome 112 as well as Firefox 112. In case of Edge and Chrome, the focus is redirected to the first menu item for the button "show reactions" and in this case saying "expanded" or "colapsed" is redundant because the user is redirected to the menu item. When the focus is not redirected, NVDA reports as expected "expanded" and "colapsed".

Closing as works for me.
